lib/application_json.rb

```diff
@@ -113,6 +113,12 @@ def applications_params
         type: 'string',
         description: 'Optional. Include only applications changed or created on
         or since a date and time. Times should be in ISO 8601 format.'
+      },
+      {
+        name: 'order',
+        type: 'string',
+        description: 'Optional. Specify how the returned applications are to be
+        sorted: "newest_first" or "oldest_first". Defaults to "newest_first".'
```

Comment:
should we use `-` instead of `_` ?
`newest-first`,`oldest-first`

Reply:
I have a slight preference for underscores here as `newest_first` and `oldest_first` are acting as symbols, ie they're for parsing by machines. To me, [hyphens (gently) imply word boundaries](https://blog.codinghorror.com/of-spaces-underscores-and-dashes/), ie they're for parsing by humans.